fe/src/pages/CREATE_DAILY_REPORT.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
```

Comment:
- for proper file naming, please use pascal case instead of camel case. e.g. `InputField`, `TableRow`, etc. [ref](https://www.theserverside.com/feature/A-guide-to-common-variable-naming-conventions)
- please add root directory for `reports`. this is for us to give room for the incoming related features like `listing`, `detail`
```
/reports
  /create
  /list
  /detail
  /edit
```